library/src/main/java/com/pokegoapi/util/Signature.java

```diff
@@ -64,7 +49,7 @@ public static void setSignature(PokemonGo api, RequestEnvelopeOuterClass.Request
 				.setSensorInfo(SensorInfo.getDefault(api))
 				.setActivityStatus(ActivityStatus.getDefault())
 				.addAllLocationFix(LocationFix.getDefault(api))
-				.setVersionCodeHash(versionCodeHash);
+				.setVersionCodeHash(-8537042734809897855L);
```

Comment:
Why fill it in? Why not leave the functionality there to xxHash64 the "whatever it actually is" and fill that part in instead so in the future it can easily be replaced by whatever it actually is.